### PR TITLE
Improve parameters name

### PR DESCRIPTION
**Pull request by ryuukk**

### CODE REVIEW

**Review by Geod24 — CHANGES_REQUESTED** [review]
Some of the names, while not great, follow what's on the man page:
- https://man7.org/linux/man-pages/man3/memchr.3.html
- https://man7.org/linux/man-pages/man3/memcmp.3.html

However, that's not the case for all of them: https://man7.org/linux/man-pages/man3/memmove.3.html

I think that we should just follow what the `man` uses, otherwise we'd have to come up with our own convention, which I'm not sure will be a productive usage of everyone's time (do we use `dst` ? `dest` ? `destination` ? etc...)

**Comment by ryuukk** [comment]
I'd personally just use ``dst`` and ``src``

Waiting to see if more people have ideas before making other changes

File: src/core/stdc/string.d [hunk 1]
```diff
@@ -35,23 +35,23 @@ nothrow:
 @nogc:
 
 ///
-inout(void)* memchr(return inout void* s, int c, size_t n) pure;
+inout(void)* memchr(return inout void* ptr, int value, size_t n) pure;
 ///
-int   memcmp(scope const void* s1, scope const void* s2, size_t n) pure;
+int   memcmp(scope const void* dest, scope const void* src, size_t n) pure;
```
Comment (n8sh) [inline src/core/stdc/string.d:40]: I do not consider this change to be an improvement. Naming the parameters "source" and "destination" implies something is moving from the source to the destination.

**Comment by RazvanN7** [comment]
@ryuukk Given that there were only oppositions to these PR, how to you plan to proceed?

**Pull request closed without merging**